stellarphot/core.py

```diff
@@ -1017,6 +1032,21 @@ def apass_dr9(field_center, radius=1 * u.degree, clip_by_frame=False, padding=10
 
     # IAU says there is a space between the acronym and the coordinates.
     raw_catalog["id"] = [f"{designation_acronym} {coord}" for coord in coord_string]
+
+    # Translate the passbands to AAVSO standard names.
+    # No need to change B and V since those are already correct.
+    # Do this *after* initialization so that the original APASS band names
+    # are used for the tidy-ification operation.
+    raw_catalog.passband_map = PassbandMap(
```

Comment:
I assume Vizier only tracks these bandpasses, so you can hardcode this, right?

Reply:
Each catalog seems to be difference 🎉 🙄 but for APASS these are the names in Vizier. A new data release of APASS could conceivably change that, but this is explicitly DR9, so a new data release would mean adding a new class (which is fine I think).